src/any_agent/serving/task_manager.py

```diff
@@ -0,0 +1,98 @@
+from collections.abc import AsyncIterable
+from typing import TYPE_CHECKING
+
+try:
```

Comment:
You could use the `with suppress(ImportError):` syntax that we adopted in the MCP code